sky/resources.py

```diff
@@ -953,6 +939,18 @@ def less_demanding_than(
                 if not self_ports <= other_ports:
                     return False
 
+        if self.disk_tier is not None:
```

Comment:
Is there a reason, `disk_tier` check needs to go after `port` check? It would be nice to have a comment for future reference.

Reply:
Originally it was for convenient (we can `return True` if it is the last field to check). Now I changed the order back 🫡